nisdc/nisemidevicecontrol.py

```diff
@@ -76,9 +76,9 @@ def __init__(self, isconfigpath):
             raise e
 
     def start(self):
-        """Starts the Instrument/Hardware sessions configured for the device control.
+        """Starts the Instrument/Hardware sessions.
         
-        Through the IS export configuration.
+        It is configured for the device control through the IS export configuration.
```

Comment:
Can we rephrase this to - Configuration for device control is done through the IS export configuration?